eo-maven-plugin/src/main/java/org/eolang/maven/TranspileMojo.java

```diff
@@ -91,7 +105,8 @@ public final class TranspileMojo extends SafeMojo {
     public void exec() throws IOException {
         final Transpiler cmp = new Transpiler(
             this.targetDir.toPath().resolve(TranspileMojo.DIR),
-            this.targetDir.toPath().resolve(TranspileMojo.PRE)
+            this.targetDir.toPath().resolve(TranspileMojo.PRE),
+            failures(this.failOnWarning)
```

Comment:
@mximp maybe it's better to move this functionality away from `Transpiler` class? We can introduce a new class `Sanitizier` (for example), which will encapsulate the path to an XML and behave like this:

```
class Sanitizer {
  void failOnErrors();
  void failOnWarnings();
}
```

the workflow of `TraspileMojo` will be this:

```
Transpiler().transpile(); // here we do XML-to-XML transformations
Sanitizer().failOnErrors();
Sanitizer().failOnWarnings();
Generator().generate(); // here we generate .java class
```

We will also split `Transpiler` into two pieces: `Transpiler` and `Generator`

Reply:
@yegor256  Quite big refactoring... do we really want to do it as part of this task?

Reply:
@mximp otherwise we'll never get back to this. The way the class is extended now is a big clumsy. That's why I suggest splitting it into pieces.